iidm/iidm-scripting/src/main/groovy/com/powsybl/iidm/network/scripting/ShuntCompensatorExtension.groovy

```diff
@@ -58,8 +58,12 @@ class ShuntCompensatorExtension {
         }
     }
 
+    /*static int getSectionCount(ShuntCompensator self) {
+        self.getSectionCount().orElseThrow(AssertionError::new) // Throws a StackOverflow exception (cyclic method call)
+    }*/
```

Comment:
@geofjamg Do you have any idea about how to prevent breaking changes in groovy for this API modification?